File: site/javascript/26.js
function giveRemainder(num){    
    let divident=Math.pow(10,(String(num).length))    
    let dividentArr=[divident]        
    let result = 0
    
    while(1){
        let localRem = divident % num
        let localQuo = Math.floor(divident / num )
        
        if (localRem==0){
            break
        }else{
            divident = localRem*10
        }
        
        if(dividentArr.includes(parseInt(divident))){
            result = dividentArr.length - dividentArr.indexOf(divident)
            break
        }else{
            dividentArr.push(divident)
        }
        
    }
    
    return result
}

let result=0
for(let i=2;i<1000;i++){
    let loaclRes = giveRemainder(i)
    result = result > loaclRes ? result : i
}

console.table(result);

File: site/python/26.py
import  math
def giveRemainder(num):
    divident = math.pow(10,len(str(num)))
    dividentArr = [divident]
    result = 0

    while 1:
        remainder = divident % num
        quotient = divident // num

        if remainder==0:
            break
        else:
            divident = remainder * 10

        if divident in dividentArr:
            result = len(dividentArr) - dividentArr.index(divident)
            break
        else:
            dividentArr.append(divident)

    return result

result = 0
for i in range(2,1001):
    ans = giveRemainder(i)
    result = result if result > ans else i

print(result)